networking.go

```diff
@@ -126,3 +126,62 @@ func startStatsdTCP(s *Server, addr *net.TCPAddr, packetPool *sync.Pool) {
 		s.ReadTCPSocket(listener)
 	}()
 }
+
+func StartSSF(s *Server, a net.Addr, tracePool *sync.Pool) {
```

Comment:
No TCP?

Reply:
I would like to focus on the thing that directly benefits us first, so we get the general shape of the change hammered out; once somebody needs TCP, I suspect it will be pretty easy to add (:

Reply:
Ok!